end-to-end-tests/cypress/pages/features.ts

```diff
@@ -1,74 +1,73 @@
 class FeaturesPage {
-
-  public slug = 'features'
+  public slug = 'features';
 
   public completeReasonForTransfer(): this {
+    // Click the first matched element
+    cy.get('[data-test="initiated"]').first().click();
 
-    cy.get('[data-test="initiated"]').click()
-
-    cy.get('h1').should('contain.text', 'What is the reason for this transfer?')
+    cy.get('h1').should('contain.text', 'What is the reason for this transfer?');
 
     // Check the labels of the radios
     cy.get('.govuk-radios__item').then(($reasons) => {
-      cy.wrap($reasons).should('have.length', 3)
-      
-      cy.wrap($reasons[0]).should('contain.text', 'Intervention')
-      cy.wrap($reasons[1]).should('contain.text', 'Initiated by trust')
-      cy.wrap($reasons[2]).should('contain.text', 'Sponsor or trust closure')
-    })
+      cy.wrap($reasons).should('have.length', 3);
```

Comment:
Don't need the semicolons, but they don't hurt so not urgent. Just a style consistency thing